app1.js

```diff
@@ -49,7 +49,12 @@ function lightsaberColor(){
         alt = "Red Lightsaber";
     }
 
-    document.write(`<img src="{url}" alt="{alt}" />`)
+    writeImageToDocument(url, alt);
+}
+
+function writeImageToDocument(imgUrl, imgAlt) {
+    let tag = `<img src="{imgUrl}" alt="{imgAlt}" />`;
+    document.write(tag);
```

Comment:
Since we're calling `document.write` from a bunch of places, it might simplify things a bit if you wrapped it up in a function. This is a good habit to get into and calls back to the conversation about functions doing 'just one thing'. It might be the case later that you end up needing to log each time you do a `document.write` or you need to check for content or maybe you want to use CSS selectors to pick where to write to instead of injecting <script> into different places in your DOM.  We'll talk about that later.